app/Http/Controllers/Api/RecitersController.php

```diff
@@ -54,11 +55,12 @@ public function index(Request $request) : JsonResponse
      */
     public function store(Request $request) : JsonResponse
     {
+        $photoName = time() . '.' . $request->image_path->getClientOriginalExtension();
+        $request->image_path->move(public_path('reciters'), $photoName);
```

Comment:
I don't think this is how I'd like to do this.

Leave this be for now, and I'll edit this branch and figure out how I think it should be done.